test/data/testSP.py

```diff
@@ -293,20 +290,11 @@ def testmoments(self):
             for m in moment32:
                 self.assertEqual(m, 0.0)
 
-    @skipForParser('ADF', 'Does not support metadata yet')
-    @skipForParser('GAMESSUK', 'Does not support metadata yet')
-    @skipForParser('Molcas', 'The parser is still being developed so we skip this test')
-    @skipForParser('Molpro', 'Does not support metadata yet')
-    @skipForParser('NWChem', 'Does not support metadata yet')
-    @skipForParser('ORCA', 'Does not support metadata yet')
-    @skipForParser('Psi3', 'Does not support metadata yet')
-    @skipForParser('Psi4', 'Does not support metadata yet')
-    @skipForParser('QChem', 'Does not support metadata yet')
-    @skipForParser('Turbomole', 'The parser is still being developed so we skip this test')
     def testmetadata(self):

```

Comment:
Do you think it makes sense to break this down into one method for each metadata attribute? Do we want to validate the value of any of the metadata?

Reply:
> Do you think it makes sense to break this down into one method for each metadata attribute?

Yes.

> Do we want to validate the value of any of the metadata?

Yes, but related to https://github.com/cclib/cclib/pull/599#pullrequestreview-149546363 and https://github.com/cclib/cclib/issues/602, testing parser-specific results is messy right now, unless we restrict it to `SP` tests.

Reply:
I think it's fine to limit to `SP` test for now in that case. Maybe sprinkle with TODOs so we now where to go from here. Anyway, we can do this incrementally.